erpnext_pk/install.py

```diff
@@ -0,0 +1,51 @@
+import frappe
+from frappe.custom.doctype.custom_field.custom_field import create_custom_fields
+
+
+
+custom_fields = {
+	# Doctype Company Mods
+	'Company': [
```

Comment:
What is the difference between a Company, and a Customer or Supplier?

Reply:
Company DocType refers to _your_ company that is using ERPNext
Customer and Suppliers are... well... customers and suppliers against which we make Sales Orders and Purchase Orders